packages/web/components/navbar/index.tsx

```diff
@@ -156,11 +156,11 @@ export const NavBar: FunctionComponent<
     // announcement banner
     const [_showBanner, setShowBanner] = useLocalStorageState(
       Announcement ? Announcement?.localStorageKey ?? "" : "",
-      true
+      null
```

Comment:
this is just not going to show the banner ever right?

You can add a banner to test this by adding the following code to line 39 in feature-flag.ts

```
  : {
    localStorageKey: "test-key",
    enTextOrLocalizationPath: "test description",
  };
  ```